README.md

```diff
@@ -4,20 +4,24 @@
 
 # UniMe - Identity Wallet
 
-Identity Wallet for people to manage Decentralized Identities and Verifiable Credentials
+Identity Wallet for people to manage Decentralized Identities and Verifiable Credentials.
 
-### Tech stack
+## Tech stack
 
-- [Tauri 2.0](https://beta.tauri.app/)
-- [SvelteKit](https://kit.svelte.dev/), [Tailwind CSS](https://tailwindcss.com/), [Melt UI](https://melt-ui.com/)
+This app is built with [Tauri 2.0](https://v2.tauri.app/). It uses [Rust](https://www.rust-lang.org/) in the back-end and [SvelteKit](https://kit.svelte.dev/), [Tailwind CSS](https://tailwindcss.com/), and [Melt UI](https://melt-ui.com/) in the front-end.
 
-## Development
+## Develop
 
-Run `npm install` in the project root to install [Prettier](https://prettier.io/). Then
+This repository uses [NPM workspaces](https://docs.npmjs.com/cli/v10/using-npm/workspaces):
```

Comment:
Shall we follow the ["offical" spelling](https://en.wikipedia.org/wiki/Npm) and use lowercase `npm` (instead of `NPM`)?

Reply:
Changed.